spec/spec.md

```diff
@@ -177,11 +180,13 @@ proofs may satisfy an input requirement.
     "input_descriptors": [
       {
         "id": "banking_input",
-        "schema": {
-          "uri": ["https://bank-standards.com/customer.json"],
-          "name": "Bank Account Information",
-          "purpose": "We need your bank and account information."
-        },
+        "name": "Bank Account Information",
+        "purpose": "We need your bank and account information.",
+        "schema": [
+          {
+            "uri": "https://bank-standards.com/customer.json"
```

Comment:
I'd suggest adding `required: true` to this "bank and account" example too -- but I'm happy either way.